sizzle/migrations/0001_initial.py

```
@@ -0,0 +1,114 @@
+# Generated by Django 4.2.24 on 2025-11-08 07:01
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.db import migrations, models
+
+from sizzle.conf import SIZZLE_ORGANIZATION_MODEL
+
+if not SIZZLE_ORGANIZATION_MODEL:
+    raise ImproperlyConfigured("SIZZLE_ORGANIZATION_MODEL must be configured before running sizzle migrations.")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)